### ISSUE TITLE
[Feature Request] Confirmation when resetting a game in progress.

### ISSUE THREAD

**Issue by derei** [SecUSo/privacy-friendly-2048#41]
The Undo and Reset are next to each other and it's only too easy to accidentally reset a game in progress.

A confirmation dialogue to avoid such accidents would be a good addition.

**Comment by Bolpat** [comment]
Frankly, the button to reset the game shouldn’t even exist in that interface. One can easily go back and select “New Game,” and that should explicitly if it’s intended.

**Comment by WAUthethird** [comment]
Agreed. Easiest option would just be to remove the button entirely.
